lib/timers.js

```diff
@@ -94,8 +94,8 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 //
 // - key = time in milliseconds
 // - value = linked list
-const refedLists = {};
-const unrefedLists = {};
+const refedLists = Object.create(null);
```

Comment:
Literals are always faster than instantiation and builder function calls, in JS and most languages.